Source/Diagnostics/BTDiagnostics.cpp

```diff
@@ -846,9 +1008,13 @@ BTDiagnostics::Flush (int i_buffer)
             m_buffer_box[i_buffer].setSmall(m_moving_window_dir, (m_buffer_box[i_buffer].smallEnd(m_moving_window_dir) + 1) );
             m_buffer_box[i_buffer].setBig(m_moving_window_dir, (m_buffer_box[i_buffer].bigEnd(m_moving_window_dir) - 1) );
             m_particles_buffer[i_buffer][0]->SetParticleBoxArray(0,vba.back());
+            for (int isp = 0; isp < m_particles_buffer.at(i_buffer).size(); ++isp) {
+                // BTD output is single level. Setting particle geometry, dmap, boxarray to level0
+                m_particles_buffer[i_buffer][isp]->SetParGDB(vgeom[0], vdmap[0], vba.back());
+            }
         }
     }
-
+    amrex::Print() << " writing data " << i_buffer <<"\n";
```

Comment:
Leftover debug statement?
```suggestion
    // amrex::AllPrint() << " writing data " << i_buffer <<"\n";
```